.github/actions/setup-poetry/action.yml

```diff
@@ -0,0 +1,14 @@
+runs:
+  using: "composite"
+  steps:
+    - name: Installing poetry
+      run: |
+        export $( \
+            PYTHONFAULTHANDLER=TRUE \
+            PYTHONUNBUFFERED=TRUE \
+            PIP_DISABLE_PIP_VERSION_CHECK=on \
+            PIP_NO_CACHE_DIR=off \
+        )
+        pip install --upgrade pip \
+        && pip install "poetry==1.0.5"
```

Comment:
How much trouble would it be to define the poetry version with a variable like it's done with the setup-python action?